src/Command/ImportScorerCommand.php

```diff
@@ -0,0 +1,184 @@
+use App\Entity\Scorer;
+use Doctrine\Common\Persistence\ObjectManager;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Security\Core\Encoder\UserPasswordEncoderInterface;
+use Symfony\Component\Stopwatch\Stopwatch;
+
+class ImportScorerCommand extends Command
+{
+    const OPTION_OVERWRITE = '--overwrite';
+    const OPTION_FORCE = '--force';
+    const ARGUMENT_PATH = 'path';
+
+    protected $encoder;
+    protected $objectManager;
+    protected $stopwatch;
+    protected $input;
+    protected $output;
+    protected $dataProvider;
+    protected $formatFactory;
+    protected $path;
+    protected $symfonyStyle;
+
+    public function __construct(UserPasswordEncoderInterface $encoder, ObjectManager $objectManager,
+                                Stopwatch $stopwatch, AdapterInterface $provider)
+    {
+        $this->encoder = $encoder;
+        $this->objectManager = $objectManager;
+        $this->stopwatch = $stopwatch;
+        $this->dataProvider = $provider;
+        parent::__construct('app:import-scorer');
+    }
+
+    /**
+     *  configure options and argument of the command.
+     */
+    protected function configure()
+    {
+        $this
+            ->setDescription('Import scorers data from a CSV file')
+            ->setHelp('CSV file HEADERS: \'username,password\'');
+        $this->addOption(
+            'force',
+            'f',
+            InputOption::VALUE_NONE,
+            'Causes data ingestion to be applied into storage'
+        );
+        $this->addOption(
+          'overwrite',
+            'o',
+            InputOption::VALUE_NONE,
+            'overwrite Scorer in database'
+        );
+        $this->addArgument(
+            'path',
+            InputArgument::REQUIRED,
+            'Source path to ingest from'
+        );
+    }
+
+    /**
+     * Initialize variables
+     *
+     * @param InputInterface $input
+     * @param OutputInterface $output
+     * @throws \Exception
+     */
+    protected function initialize(InputInterface $input, OutputInterface $output)
+    {
+        $this->input = $input;
+        $this->output = $output;
+        $this->path = $input->getArgument(self::ARGUMENT_PATH);
+        $this->symfonyStyle = new SymfonyStyle($input, $output);
+    }
+
+    /**
+     * import scorer from data file to the database
+     *
+     * @param InputInterface $input
+     * @param OutputInterface $output
+     * @return int
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $this->stopwatch->start('import');
+        $this->symfonyStyle->title('Human-Scoring Import');
+        if (!$input->hasParameterOption(self::OPTION_OVERWRITE)) {
```

Comment:
Use `getOption` instead of `hasParameterOption`. It's not reliable as it is stated in its documentation.